app/models/container_project.rb

```diff
@@ -26,6 +28,7 @@ class ContainerProject < ApplicationRecord
   virtual_column :routes_count,      :type => :integer
   virtual_column :replicators_count, :type => :integer
   virtual_column :containers_count,  :type => :integer
+  virtual_column :images_count,      :type => :integer
 
   def groups_count
```

Comment:
sure, make it in a followup pr. but no, it is not purely aesthetic. It helps us avoid n+1 queries.